--- a/configfsf.sub
+++ b/configfsf.sub
@@ -1759,7 +1759,7 @@
 	     | onefs* | tirtos* | phoenix* | fuchsia* | redox* | bme* \
 	     | midnightbsd* | amdhsa* | unleashed* | emscripten* | wasi* \
 	     | nsk* | powerunix* | genode* | zvmoe* | qnx* | emx* | zephyr* \
-	     | fiwix* | mlibc* )
+	     | fiwix* | mlibc* | ohos* )
 		;;
 	# This one is extra strict with allowed versions
 	sco3.2v2 | sco3.2v[4-9]* | sco5v6*)
--- a/Makefile.am
+++ b/Makefile.am
@@ -113,7 +113,7 @@
 LIBGMPXX_LT_AGE      = 7
 
 
-SUBDIRS = tests mpn mpz mpq mpf printf scanf rand cxx demos tune doc
+SUBDIRS = tests mpn mpz mpq mpf printf scanf rand cxx demos tune
 
 EXTRA_DIST = configfsf.guess configfsf.sub .gdbinit INSTALL.autoconf \
 	     COPYING.LESSERv3 COPYINGv2 COPYINGv3
--- a/Makefile.in
+++ b/Makefile.in
@@ -573,7 +573,7 @@
 LIBGMPXX_LT_CURRENT = 11
 LIBGMPXX_LT_REVISION = 0
 LIBGMPXX_LT_AGE = 7
-SUBDIRS = tests mpn mpz mpq mpf printf scanf rand cxx demos tune doc
+SUBDIRS = tests mpn mpz mpq mpf printf scanf rand cxx demos tune
 
 # Put asl.h here for now.
 
